test/SpecHelper.hs

```diff
@@ -73,26 +67,20 @@ withApp config perform = do
 
   where middle = defaultMiddle
 
-
-resetDb :: IO ()
-resetDb = do
-  pool :: H.Pool P.Postgres
-    <- H.acquirePool pgSettings testPoolOpts
-  void . liftIO $ H.session pool $
-    H.tx Nothing $ do
-      H.unitEx [H.stmt| drop schema if exists test cascade |]
-      H.unitEx [H.stmt| drop schema if exists private cascade |]
-      H.unitEx [H.stmt| drop schema if exists postgrest cascade |]
-
+setupDb :: IO ()
+setupDb = do
+  void $ readProcess "psql" ["-a", "-f", "test/fixtures/database.sql"] []
```

Comment:
On my mac it defaults to trying to connect to a database named after my username, which fails:

```
postgrest-0.3.0.1: test (suite: spec)
Progress: 1/2psql: FATAL:  database "j" does not exist
spec: readCreateProcess: psql "-a" "-f" "test/fixtures/database.sql" (exit 2): failed
Test suite failure for package postgrest-0.3.0.1
    spec:  exited with: ExitFailure 1
Logs printed to console
```

If you add `-d postgres` it should be OK assuming we can count on that database existing for all postgres installations.